Makefile

```diff
@@ -3,8 +3,12 @@ export COMPOSE_DOCKER_CLI_BUILD=1
 
 .EXPORT_ALL_VARIABLES: ;
 
-OB_APPS=developer-tpp financroo-tpp consent-page consent-self-service-portal consent-admin-portal bank
+OB_APPS=developer-tpp financroo-tpp consent-page-cdr consent-self-service-portal consent-admin-portal bank
```

Comment:
may be we should remove this consent-page-cdr from here

Reply:
yes, it should be just `consent-page`